components/Impact/BlogCard.js

```diff
@@ -0,0 +1,34 @@
+import Link from 'next/link';
+import React from 'react';
+
+import styles from '../../styles/pages/impact.module.scss';
+
+export default function BlogCard({title, author, date, length, img, href, authorImg, authorAlt, backgroundStyle}) {
+	// stylelint's CSS-in-JS is giving me an error here
+	/* stylelint-disable */
+	// const backgroundStyle = { backgroundImage: `url("${img}")`};
+	/* stylelint-enable */
+
+	return (
+		// TODO: not sure how, but we need to semantically
+		// insert a description of the image for screenreaders;
+		// not natively supported by background-image
```

Comment:
Assuming this has been taken care of using the `alt` property of the `<img>` tag here.

Reply:
No, so part of the problem is that `backgroundStyle` encodes the image as a `background-image` for the article itself. This is not amazing, since there's no way to properly encode alt-text into `background-*` based properties (or at least to my knowledge).

This is _technically_ fine since the images are decorative anyways, but a later pass might involve adding alt-text via an `.sr-only` type class.